### PR TITLE
Create witness.md

### PR DESCRIPTION
**Pull request by Amrinder-Singh29**
Adding a comment to ensure that storage endpoint is whitelisted as well along with port 443

### CODE REVIEW

**Comment by PRMerger7** [comment]
@Amrinder-Singh29 : Thanks for your contribution! The author(s) have been notified to review your proposed change. @v-dasis

**Comment by ktoliver** [comment]
Hi @JasonGerend - The listed author `v-dasis` isn't found. Could you review or reassign the pull request, and, if needed, update the article metadata in a pull request in the private repo? Thanks!

File: azure-stack/hci/manage/witness.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Setting up a witness resource is highly recommended for all clusters, and should
 
 You can either use an SMB file share as a witness or an Azure cloud witness. An Azure cloud witness is recommended, provided all server nodes in the cluster have a reliable internet connection. This article covers creating a cloud witness.
 
-Cloud witness uses the HTTPS default port 443 to establish outbound communication with the Azure blob service. Ensure that the HTTPS port is accessible.
+Cloud witness uses the HTTPS default port 443 to establish outbound communication with the Azure blob service. Ensure that the HTTPS port is accessible. Azure uses **.core.windows.net** as the Endpoint. Hence you need to have it included in any firewall allow lists you're using between the cluster and Azure Storage.
```
Comment (JasonGerend) [inline azure-stack/hci/manage/witness.md:19]:
```suggestion
Cloud witness uses the HTTPS default port 443 to establish outbound communication with the Azure Blob Storage service. Make sure that port 443 is open in your firewalls and that `*.core.windows.net` is included in any firewall allow lists you're using between the cluster and Azure Storage.
```

Comment (Amrinder-Singh29) [reply azure-stack/hci/manage/witness.md:19]: Thanks @JasonGerend. Yes the changes looks good

**Comment by PRMerger-2** [comment]
@Amrinder-Singh29 : Thanks for your contribution! The author(s) have been notified to review your proposed change. @v-dasis

**Review by JasonGerend — APPROVED** [review]

**Comment by JasonGerend** [comment]
#sign-off

**Pull request merged**